docs/sidebars.js

```diff
@@ -61,7 +61,18 @@
              "user-guides/centauri-transfers",
            ],
          },
- 
+         {
+           type: "category",
+           label: "FAQs",
+           link: {
+             type: "generated-index",
+             slug: "faqs",
+           },
+           collapsible: false,
+           items: [
+             "trustless-faq",
```

Comment:
this should have the correct path. in the other pr, only the title is renamed, not the file, so the correct path here should be "user-guides/faq"

Reply:
<img width="1737" alt="Screenshot 2023-09-12 at 23 43 15" src="https://github.com/ComposableFi/composable/assets/108475490/3c486868-0b02-40a3-904d-0fc802afdbc4">
the file is also renamed, but yep, the path is wrong, will change to "user-guides/trustless-faq"